src/modules/poweraccent/PowerAccent.Core/Languages.cs

```diff
@@ -423,6 +427,16 @@ private static string[] GetDefaultLetterKeyDE(LetterKey letter)
             };
         }
 
+        // Hebrew
+        private static string[] GetDefaultLetterKeyHE(LetterKey letter)
```

Comment:
Is it a complete set of accents for the Hebrew language?
If not, then please consider either adding these symbols to GetDefaultLetterKey or adding a complete set of Accents for the Hebrew language.

Reply:
I have added some additional less common letters and punctuations, which are unique to Hebrew script. 
Is it worth adding vowels as well?

Reply:
Yes, if they are accented.